issues/payment-handler-missingtask.md

```diff
@@ -0,0 +1,15 @@
+---
+Title: Missing tasks in parallel steps in Payment Handler API
+Tracked: N/A
+Repo: 'https://github.com/w3c/payment-handler'
+---
+
+While crawling [Payment Handler API](https://w3c.github.io/payment-handler/), the following algorithms fire an event, or resolve or reject a Promise, within a step that runs [in parallel](https://html.spec.whatwg.org/multipage/infrastructure.html#in-parallel) without first queuing a task:
+* [ ] The algorithm that starts with "Upon receiving a PaymentRequest by way of PaymentRequest.show() and subsequent user selection of a payment handler, the user agent MUST run the following steps:" resolves/rejects a promise directly in a step that runs in parallel
+* [ ] An algorithm resolves/rejects a promise directly in a step that runs in parallel
```

Comment:
```suggestion
* [ ] The [Open Window Algorithm](https://w3c.github.io/payment-handler/#open-window-algorithm) resolves/rejects a promise directly in a step that runs in parallel
```

@tidoust, I've noticed a number of times where there was no link for the algorithm - wouldn't it make sense to at least link to the containing section in such a situation?